sdk/storage/azure_storage_blob/tests/blob_container_client.rs

```diff
@@ -129,116 +142,116 @@ async fn test_list_blobs(ctx: TestContext) -> Result<(), Box<dyn Error>> {
     Ok(())
```

Comment:
Relevant test case that I refactored to now work against the newly returned `Pager`, but this is hitting an error:
>Error: Error { context: Custom(Custom { kind: DataConversion, error: Error("expected value", line: 1, column: 1) }) }